src/ConversionTable.tsx

```diff
@@ -26,54 +31,49 @@ const ConversionTable = ({
-                    </td>
-                </tr>
-                <tr aria-label="Currency conversion">
-                    {/* TODO: Look up what the best practice is for displaying a string across mulitple table cells. This trailing space feels hacky, but
-                    if I don't include it, it reads like "÷ 1USD per USD" */}
-                    <td className="operation">{currencyExchangeFormula.operation} {Number(currencyExchangeFormula.rate).toFixed(7)} </td>
-                    <td className="operation-description">
-                        {currencyExchangeFormula.operation === "÷" ? `${sourceCurrency} per ${targetCurrency}` : `${targetCurrency} per ${sourceCurrency}`}
-                        <br />
-                        <em>(updated {targetCurrencyUpdatedDate})</em>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
+            setShowDetails(newValue)
+        }}>{showDetails ? "Hide full conversion details..." : "Show full conversion details..."}</button>
+        <ul className={`details ${showDetails ? "visible" : ""}`} aria-hidden={!showDetails} aria-label="Conversion Details">
+            <li style={{ "--i": 0 } as React.CSSProperties}>
+                <label id="cost-label">Cost</label>
+                <span aria-labelledby="cost-label">{topNumber} {topCurrency} per {topUnit}</span>
+            </li>
+            <li style={{ "--i": 1 } as React.CSSProperties}>
+                <label id="currency-conversion-rate-label">Currency conversion rate</label>
+                <span aria-labelledby="currency-conversion-rate-label">
+                    {sourceCurrencyAbsoluteCost > targetCurrencyAbsoluteCost ?
+                        `${sourceCurrencyAbsoluteCost / targetCurrencyAbsoluteCost} ${topCurrency} = 1 ${bottomCurrency}` :
+                        `1 ${topCurrency} = ${targetCurrencyAbsoluteCost / sourceCurrencyAbsoluteCost} ${bottomCurrency}`}
+                </span><br />
+                <em>(Last updated: {bottomCurrencyUpdatedDate})</em>
+            </li>
+            <li style={{ "--i": 2 } as React.CSSProperties}>
+                <label id="volume-conversion-rate-label">Volume conversion rate</label>
+                <span aria-labelledby="volume-conversion-rate-label">
+                    {targetVolumeInLiters > sourceVolumeInLiters ?
+                        `${targetVolumeInLiters / sourceVolumeInLiters} ${topUnit}s = 1 ${bottomUnit}` :
+                        `1 ${topUnit} = ${sourceVolumeInLiters / targetVolumeInLiters} ${bottomUnit}s`}
+                </span>
+            </li>
+            <li style={{ "--i": 3 } as React.CSSProperties}>
+                <label id="converted-cost-label">Converted cost</label>
+                <span aria-labelledby="converted-cost-label">{bottomNumber} {bottomCurrency} per {bottomUnit}</span>
+            </li>
```

Comment:
Feels like this could be handled more elegantly in an array that gets mapped instead of repeating all this syntax. The `id`s could be generated dynamically, and the hard-coded `--i: 3` stuff is unattractive